src/lib/hash.c

```diff
@@ -163,6 +163,11 @@ pgp_hash_name_botan(pgp_hash_alg_t hash)
         return "SM3";
 #endif
 
+#if defined(BOTAN_HAS_CRC24)
```

Comment:
I'm going to abolish implementation of this function in next commit.

Reply:
Nice one @flowher !

Reply:
No. But we don't have control over that (as for any other algorithm).